test/e2e/tests/notifications/account-syncing/sync-after-modifying-account-name.spec.ts

```diff
@@ -90,29 +89,24 @@ describe('Account syncing @no-mmi', function () {
             NOTIFICATIONS_TEAM_PASSWORD,
           );
 
-          await driver.clickElement('[data-testid="account-menu-icon"]');
-
-          await driver.wait(async () => {
-            const internalAccounts = await driver.findElements(
-              '.multichain-account-list-item',
-            );
-            return (
-              internalAccounts.length ===
-              userStorageMockttpController.paths.get('accounts')?.response
-                .length
-            );
-          }, 20000);
-
-          await driver.wait(async () => {
-            const internalAccounts = await driver.findElements(
-              '.multichain-account-list-item .multichain-account-list-item__account-name',
-            );
-            const lastAccountName = await internalAccounts[
-              internalAccounts.length - 1
-            ].getText();
+          const header = new HeaderNavbar(driver);
+          await header.check_pageIsLoaded();
+          await header.openAccountMenu();
 
-            return lastAccountName === 'My third account';
-          }, 20000);
+          const accountListPage = new AccountListPage(driver);
+          await accountListPage.check_pageIsLoaded();
+          await accountListPage.check_numberOfAvailableAccounts(
+            accountsSyncMockResponse.length,
+          );
+          await accountListPage.check_accountIsNotDisplayedInAccountList(
```

Comment:
just out of curiosity, why do we have one method for "isNotDisplayed" and one method for "isDisplayed"? can not be implemented in a single method where it handles the exception in case it is not displayed and returns false?

Reply:
This is a good point actually. I just used the method as it was implemented. Also, from a lot of the other tests I have seen, it looks like the `check_` methods are quite explicit. 

@chloeYue might have some insight?

Reply:
Yes we try to make `check_` methods explicit and easy to use, so we can call the `check_` methods directly, and we don't need to assert in test body like `assert.equal(accountListPage.check_accountIsNotDisplayedInAccountList(xxx), false);`